src/api/trace/b3_propagator.rs

```diff
@@ -28,6 +30,12 @@ static B3_SPAN_ID_HEADER: &str = "x-b3-spanid";
 static B3_SAMPLED_HEADER: &str = "x-b3-sampled";
 static B3_PARENT_SPAN_ID_HEADER: &str = "x-b3-parentspanid";
 
+lazy_static::lazy_static! {
+static ref B3_SINGLE_FIELDS: [String; 1] = [B3_SINGLE_HEADER.to_string()];
+static ref B3_MULTI_FIELDS: [String; 4] = [B3_TRACE_ID_HEADER.to_string(), B3_SPAN_ID_HEADER.to_string(), B3_SAMPLED_HEADER.to_string(), B3_DEBUG_FLAG_HEADER.to_string()];
+static ref B3_SINGLE_MULTI_FIELDS: [String; 5] = [B3_SINGLE_HEADER.to_string(), B3_TRACE_ID_HEADER.to_string(), B3_SPAN_ID_HEADER.to_string(), B3_SAMPLED_HEADER.to_string(), B3_DEBUG_FLAG_HEADER.to_string()];
```

Comment:
Maybe `B3_SINGLE_AND_MULTI_FIELDS`? or `B3_ALL_FIELDS`? SINGLE_MULTI just sounds a little contradictory.

Reply:
Also these static refs could be indented in

Reply:
Sounds reasonable. Will change those.